config_templates/gretel/navft/navft-tabular.yml

```diff
@@ -0,0 +1,11 @@
+# Default configuration for NAVFT.
+
+schema_version: "1.0"
+name: "navft-tabular" 
+models:
+  - navft:
```

Comment:
I assumed we'd use `navft` as the model name here instead of `atlas`. Is that the right external facing name?